.eslintrc.js

```diff
@@ -4,4 +4,23 @@ require('@uniswap/eslint-config/load')
 
 module.exports = {
   extends: '@uniswap/eslint-config/react',
+  overrides: [
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      rules: {
+        'no-restricted-imports': [
```

Comment:
If there were to be an existing definition for `no-restricted-imports` in `@uniswap/eslint-config/react`, would this overwrite it or extend it? I think it would overwrite it. Is there a way to instead extend it?

Reply:
<img width="557" alt="image" src="https://user-images.githubusercontent.com/66155195/228628312-dd5ae6e7-7fb4-4fd4-99c7-48fcbe71119b.png">

this merges the rules so we're good i think

Reply:
currently `@uniswap/eslint-config/react` uses [a different rule](https://github.com/Uniswap/eslint-config/blob/main/react.js#L28), `@typescript-eslint/no-restricted-imports`, so it may not be merged. This may lead to problems in the future.

Reply:
afaict there is no way to do this - if you want to extend vs override you need to bring in the old definition and redefine it at the override level. :(